.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
nickpick: Just curious. How do you come out with this folder name. Is it the husky standard?

Reply:
Yep this is automatically added to the start of every bash file that Husky manages and so if we don't include it then Husky won't work correctly